### PR TITLE
Implement OTAR projects widget

### PR DESCRIPTION
**Pull request by mirandaio**
Closes https://github.com/opentargets/platform/issues/1549

### CODE REVIEW

File: src/sections/disease/OTProjects/index.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export const definition = {
+  id: 'otarProjects',
+  name: 'Open Targets Projects',
+  shortName: 'OP',
+  // TODO: determine if there's data using real data from API
+  hasData: () => true,
```
Comment (LucaFumis) [inline src/sections/disease/OTProjects/index.js:6]: So for the moment we use the data from the json, and will update to API at a later stage?

Comment (mirandaio) [reply src/sections/disease/OTProjects/index.js:6]: @LucaFumis yes, that's right

**Review by LucaFumis — APPROVED** [review]
Looks good, assuming that for now we are using the data from the JSON and will later update to the API?

File: src/sections/disease/OTProjects/index.js (hunk 1 above)
Comment (LucaFumis) [reply src/sections/disease/OTProjects/index.js:6]: Great! thanks @mirandaio

**Pull request merged**